<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>665</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Delilah</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>665</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionNewProcess"/>
     <addaction name="actionNewQueue"/>
     <addaction name="actionNewOperation"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionQuitApplication"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewProcess"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="sceneToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelect"/>
   <addaction name="actionNewQueue"/>
   <addaction name="actionNewOperation"/>
   <addaction name="actionConnect"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionNewProcess">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/new_process.png</normaloff>:/icons/new_process.png</iconset>
   </property>
   <property name="text">
    <string>Process</string>
   </property>
   <property name="toolTip">
    <string>Create New Process</string>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="actionNewQueue">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/new_queue.png</normaloff>:/icons/new_queue.png</iconset>
   </property>
   <property name="text">
    <string>Queue</string>
   </property>
   <property name="toolTip">
    <string>Insert New Queue</string>
   </property>
  </action>
  <action name="actionQuitApplication">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionNewOperation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/new_operation.png</normaloff>:/icons/new_operation.png</iconset>
   </property>
   <property name="text">
    <string>Operation</string>
   </property>
   <property name="toolTip">
    <string>Add Operation</string>
   </property>
  </action>
  <action name="actionSelect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/move.png</normaloff>:/icons/move.png</iconset>
   </property>
   <property name="text">
    <string>select</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resources/icons.qrc">
     <normaloff>:/icons/connection.png</normaloff>:/icons/connection.png</iconset>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="toolTip">
    <string>Connect Queue with Operation</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources/icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionNewProcess</sender>
   <signal>activated()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>createNewWorkspace()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tabWidget</sender>
   <signal>tabCloseRequested(int)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>removeTab(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>276</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createTab()</slot>
  <slot>removeTab(int)</slot>
  <slot>addQueue()</slot>
  <slot>createNewWorkspace()</slot>
 </slots>
</ui>
